Blockzilla/BrowserViewController.swift

```diff
@@ -942,6 +953,7 @@ extension BrowserViewController: URLBarDelegate {
     func urlBarDidLongPress(_ urlBar: URLBar) {
         let customURLItem = PhotonActionSheetItem(title: UIConstants.strings.customURLMenuButton, iconString: "icon_link") { action in
             urlBar.addCustomURL()
+            UserDefaults.standard.set(false, forKey: TipManager.TipKey.autocompleteTip)
```

Comment:
I feel like it would be nice to have `TipManager` take care of setting the `UserDefaults` value. `BrowserViewController` shouldn't have to know that `TipManager` uses `UserDefaults`